lib/install-utils.js

```diff
@@ -66,20 +67,21 @@ const chmod = (where) =>
 async function setDriverFilePermissions(where) {
   debug('setDriverFilePermissions', where);
 
-  const requireChmod = await new Promise((resolve) =>
-    fs.access(where, fs.R_OK | fs.X_OK, (err) => {
-      if (err) {
-        debug('error in fs.access', where, err);
-      }
-      resolve(!!err);
-    })
-  );
+  const requireChmod = await gotAccess(where)
 
   if (requireChmod) {
     await chmod(where);
   }
 }
 
+async function gotAccess(where) {
```

Comment:
This function will always return `undefined`, let's make sure we return something 😉

Reply:
I added returned value. it works on my machine.